mixer/adapter/list/config/adapter.list.config.pb.html

```diff
@@ -32,7 +32,7 @@ <h2 id="Params">Params</h2>
 <td><code>providerUrl</code></td>
 <td><code>string</code></td>
 <td>
-<p>Where to find the list to check against. This may be ommited for a completely local list.</p>
+<p>Where to find the list to check against. This may be omitted for a completely local list.</p>
```

Comment:
Fixes in the various .pb.html files need to be reflected in the original .proto files. the .pb.html rile is auto-generated from the .proto.

Reply:
sorry, little knowledge about it:wink:
In this condition, i would like add `.pb.html` to `bin/.spelling_failure` file. If i wrong, please correct me

Reply:
Done, fixed in last commit. thanks